src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -37,6 +37,9 @@ import * as DOM from 'vs/base/browser/dom';
 import { onUnexpectedError } from 'vs/base/common/errors';
 import { IAccessibilityService } from 'vs/platform/accessibility/common/accessibility';
 import { IQuickInputService } from 'vs/platform/quickinput/common/quickInput';
+import { localize } from 'vs/nls';
+
+const cellWithNullMessage = localize('editData.cellWithNullMessage', "This cell contains the Unicode null character which is currently not supported for editing.");
```

Comment:
does SSMS also show this error?

Reply:
Not sure if it matters if SSMS has this check, unless we have a reason to believe it could never occur.  If there are existing null characters in the string they will be changed to newlines, which is incorrect.